### PR TITLE
Update furret.json

### PR DESCRIPTION
**Pull request by furretra1n**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/furret.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "email": "furret@transistorcafe.net"
     },
     "record": {
-        "CNAME": "furret.tplinkdns.com"
+        "CNAME": "walccdev.github.io"
```
Comment (DEV-DIBSTER) [inline domains/furret.json:7]: Nothing is displayed.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/furret.json (hunk 1 above)
Comment (furretra1n) [reply domains/furret.json:7]: its a blank page, making the site

Comment (DEV-DIBSTER) [reply domains/furret.json:7]: When the site is finished, you may open a pull request and then it will get approved.

Comment (furretra1n) [reply domains/furret.json:7]: finished

**Comment by furretra1n** [comment]
finished the site yay

**Comment by furretra1n** [comment]
Mon Nov  6 07:50:39 PM EST 2023

**Review by wdhdev — APPROVED** [review]

**Pull request merged**